catalog/RUB/RUB_RUBCLIM_LCZ_global_lcz_map_latest.jsonnet

```diff
@@ -0,0 +1,345 @@
+            description: 'Sparsely built',
+          },
+          {
+            value: 10,
+            color: '555555',
+            description: 'Heavy industry',
+          },
+          {
+            value: 11,
+            color: '006a00',
+            description: 'Dense Trees (LCZ A)',
+          },
+          {
+            value: 12,
+            color: '00aa00',
+            description: 'Scattered Trees (LCZ B)',
+          },
+          {
+            value: 13,
+            color: '648525',
+            description: 'Bush, scrub (LCZ C)',
+          },
+          {
+            value: 14,
+            color: 'b9db79',
+            description: 'Low plants (LCZ D)',
+          },
+          {
+            value: 15,
+            color: '000000',
+            description: 'Bare rock or paved (LCZ E)',
+          },
+          {
+            value: 16,
+            color: 'fbf7ae',
+            description: 'Bare soil or sand (LCZ F)',
+          },
+          {
+            value: 17,
+            color: '6a6aff',
+            description: 'Water (LCZ G)',
+          },
+        ],
+      },
+      {
+        name: 'LCZ_Probability',
+        description: |||
+            A probability layer that identifies how often the modal LCZ was
+            chosen per pixel (e.g. a probability of 60% means that the modal
+            LCZ class was mapped 30 times out of 50 LCZ models). This is a
+            pixel-based probability derived from the LCZ layer
+        |||,
+        'gee:units': units.percent,
+      },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'LCZ',
+        lookat: {
+          lat: 40.72,
+          lon: -73.99,
+          zoom: 10,
+        },
+        image_visualization: {
+          band_vis: {
+            min: [
+              1,
+            ],
+            max: [
+              17,
+            ],
+            palette: [
+              '8c0000',
+              'd10000',
+              'ff0000',
+              'bf4d00',
+              'ff6600',
+              'ff9955',
+              'faee05',
+              'bcbcbc',
+              'ffccaa',
+              '555555',
+              '006a00',
+              '00aa00',
+              '648525',
+              'b9db79',
+              '000000',
+              'fbf7ae',
+              '6a6aff'
+            ],
+            bands: [
+              'LCZ_Filter',
+            ],
+          },
+        },
+      },
+    ],
+  },
+  'sci:doi': '10.5281/zenodo.6364593',
+  'sci:citation': |||
+    Demuzere M.; Kittner J.; Martilli A.; Mills, G.; Moede, C.; Stewart, I.D.; van Vliet, J.; Bechtel, B.
+    A global map of local climate zones to support earth system modelling and urban-scale environmental science.
+    Earth System Science Data 2022, 14 Volume 8: 3835-3873.
+    [doi:10.5194/essd-14-3835-2022](https://doi.org/10.5194/essd-14-3835-2022)
+  |||,
+  'gee:terms_of_use': ee.gee_terms_of_use(license),
+  'gee:user_uploaded': true,
```

Comment:
please temporarily also add
"gee:skip_indexing": true

We need to perform one more step internally to activate the dataset.

Reply:
Ok, I have added this in the file.